ocaml/idl/datamodel_pool.ml

```diff
@@ -497,6 +497,14 @@ open Datamodel_types
       ~allowed_roles:_R_POOL_OP
       ()
 
+  let assert_can_enable_tls_verification = call
+      ~flags:[`Session]
+      ~lifecycle:[Published, rel_next, ""]
+      ~name:"assert_can_enable_tls_verification"
+      ~doc:"Check that TLS verification can be enabled"
+      ~allowed_roles:_R_READ_ONLY
+      ()
+
```

Comment:
Please remove this commit (3ae3f0aa39264ea805fff7a0a710b7c2ba5cc505) It's not needed anymore